feat: add entry about IT (#3052)
Codecademy/docs: content/general/information-technology/information-technology.md

---
Title: 'Information Technology'
Description: 'Information Technology is the use of digital technological tools to store and turn data into useful information'
Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/cybersecurity'
CatalogContent:
  - 'introduction-to-information-technology'

> ---

Reviewer: You still need to format the Metadata as a term entry, it's still laid out as a topic.

> <link rel="cannonical" href="https://www.codecademy.com/courses/introduction-to-it/articles/it-overview"/>

Reviewer: Don't use HTML in the Markdown.

> **Information Technology**, also known as IT, is the use of digital technological tools to store and turn data into useful information. Since we live in a technology-driven world, IT plays a major role in our lives whether its fixing a cracked screen on a phone, doing a data transfer from an old computer to another one, or repairing a PC's hardware.

Reviewer: Can we rewrite this without using "we" or "our"?

## Roles

> These are the common roles in the IT industry:

Reviewer: ```suggestion
These are some common roles in the IT industry:
```

- Hardware Technicians: Installs, repairs, and maintains [computer hardware](https://www.codecademy.com/resources/docs/general/computer-hardware). May also be required to troubleshoot hardware and software issues, as well as provide training and support to users on how to use the devices effectively.

> - Network Engineers: Designs,plans, and implements network infrastructures ranging from local area networks(LANs), wide area networks(WANs), and intranets for businesses and organizations. Also analyzes network traffic and discuss suggestions how to increase it to fulfill the organization/business' needs.

Reviewer: ```suggestion
- Network Engineers: Designs, plans, and implements network infrastructures ranging from local area networks(LANs), wide area networks(WANs), and intranets for businesses and organizations. Also analyzes network traffic and discusses suggestions how to optimize it to fulfill the organization/business' needs.
```

> - Desktop Support Personnel: Typically provides tech support(e.g.,restoring and backing up data to mobile devices) to customers via computer or phone.

Reviewer: ```suggestion
- Desktop Support Personnel: Typically provides tech support (e.g. restoring and backing up data to mobile devices) to customers via computer or phone.
```
Can we include more than one example, they do more than restoring/backing up data.

## Certifications

> Some IT roles require applicants to become certified. Common types of certifications include:

Reviewer: No Microsoft certifications? Should include at lest the most popular one or two.

---
Title: 'Information Technology'
Description: 'Information Technology is the use of digital technological tools to store and turn data into useful information'
Subjects:

>   - 'Information Technology'

Author: The documentation mentioned that I can add a new one @SSwiniarski.

>   - 'Information Technology'

Reviewer: > The documentation mentioned that I can add a new one @SSwiniarski.

@CBID2 Then you need to add the term to the subjects file in this PR.

  - 'Networking'
Tags:

>   - 'information technology'

Author: The documentation mentions that I can add a new one @SSwiniarski.

>   - 'information technology'

Reviewer: > The [documentation](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) mentions that I can add a new one @SSwiniarski

Then you need to add the term to the Tags file in this PR. (And capitalize it.)

CatalogContent:
  - 'introduction-to-information-technology'
  - 'paths/fundamentals-of-cybersecurity'
---

> **Information Technology**, also known as IT, is the use of digital technological tools to store and turn data into useful information. Due to living in a technology-driven world, IT plays a major role in people's lives whether it's fixing a cracked screen on a phone, doing a data transfer from an old computer to another one, or repairing a PC's hardware.

Reviewer 2: I think we should broaden the definition a little bit more. Feel free to change the wording. 
```suggestion
**Information Technology**, also known as IT, is the use of digital tools and technologies to store, process, obtain, transmit, and manage useful information. IT encompasses a spectrum of tasks, including designing, implementing, and maintaining systems, networks, databases, and applications. Due to living in a technology-driven world, IT plays a major role in people's lives whether it's fixing a cracked screen on a phone, doing a data transfer from an old computer to another one, or repairing a PC's hardware.
```

## Roles

These are some common roles in the IT industry:

- Hardware Technicians: Installs, repairs, and maintains [computer hardware](https://www.codecademy.com/resources/docs/general/computer-hardware). May also be required to troubleshoot hardware and software issues, as well as provide training and support to users on how to use the devices effectively.
- Network Engineers: Designs, plans, and implements network infrastructures ranging from local area networks(LANs), wide area networks(WANs), and intranets for businesses and organizations. Also analyzes network traffic and discusses suggestions how to optimize it to fulfill the organization/business' needs.

> - Desktop Support Personnel: Typically provides tech support (e.g. restoring and backing up data to mobile devices) to customers via computer or phone.

Reviewer 2: ```suggestion
- Desktop Support Personnel: Typically provides tech support (e.g. restoring and backing up data to mobile devices) to customers via computer or phone. Handles a range of technical issues, including hardware and software problems.
```
